IdentityCore/src/controllers/broker/MSIDSSOExtensionInteractiveTokenRequestController.h

```diff
@@ -25,7 +25,6 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-API_AVAILABLE(ios(13.0), macos(10.15))
```

Comment:
Minimum version for iOS is now 14.0 (140000) and for macOS is 10.15 (101500) so all checks like this have been removed as they are now always true